shepherd_bms/src/main.cpp

```diff
@@ -95,7 +95,7 @@ bool chargingCheck(AccumulatorData_t *bmsdata)
 	static Timer chargeTimeout;
 
 	if(!chargeTimeout.isTimerExpired()) return false;
-	if(!compute.isCharging()) return false;
+	if(!compute.isChargingEnabled() | bmsdata->packCurrent < .7) return false;
```

Comment:
guessing this is supposed to be logical or || not bitwise | ? or am i missing something